tests/cases/fourslash/quickInfoForConstTypeReference.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference path="fourslash.ts" />
+
+////"" as /**/const;
+
+verify.quickInfoAt("", "");
```

Comment:
Use `verify.not.quickInfoExists()` instead.

Reply:
updated!